### PR TITLE
add audit report

### PR DESCRIPTION
**Pull request by cychuang0924**
**Before submitting a pull request**, please make sure the following is done:

### Checklist

- [x] Fork [the repository](https://github.com/getamis/alice) and create your new branch from `master`.
- [x] Please mention the name of community in Pull Request title.
- [x] Git Commit Messages
    - [x] Use the present tense (Also in Pull Request title): "Add feature" not "Added feature"
    - [x] Use the imperative mood (Also in Pull Request title): "Move cursor to..." not "Moves cursor to..."
    - [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,11 +21,22 @@ For example, an important contract not only requires enough signatures, but also
...
 > One of references of HTSS is [A Hierarchical Threshold Signature](https://medium.com/getamis/a-hierarchical-threshold-signature-830683b87873) or See [Example](#Example).
 
 
+## Audit Report:
+Alice has been audited by [Kudelski Security](https://www.kudelskisecurity.com). The details can be found in [here](REPORT_2020-05-19_FINAL_CORRECTED.pdf).
+
+
+## Warning:
+
+Although Alice has been audited, you  should still be careful to use it.
```
Comment (markya0616) [inline README.md:34]: "you  should" -> "you should"

File: README.md [hunk 2]
```diff
@@ -21,11 +21,22 @@ For example, an important contract not only requires enough signatures, but also
...
 
 
+## Audit Report:
+Alice has been audited by [Kudelski Security](https://www.kudelskisecurity.com). The details can be found in [here](REPORT_2020-05-19_FINAL_CORRECTED.pdf).
+
+
+## Warning:
+
+Although Alice has been audited, you  should still be careful to use it. 
+1. Using  end-to-end encryption to transfer message between two parts is necessary.
```
Comment (markya0616) [inline README.md:35]: "Using  end-to-end" -> "Using end-to-end"

Comment (markya0616) [reply README.md:35]: "two parts" -> "two parties"?

File: README.md [hunk 3]
```diff
@@ -21,11 +21,22 @@ For example, an important contract not only requires enough signatures, but also
...
 
+## Audit Report:
+Alice has been audited by [Kudelski Security](https://www.kudelskisecurity.com). The details can be found in [here](REPORT_2020-05-19_FINAL_CORRECTED.pdf).
+
+
+## Warning:
+
+Although Alice has been audited, you  should still be careful to use it. 
+1. Using  end-to-end encryption to transfer message between two parts is necessary. 
+2. If any error message occur during execution Alice, you should stop and restart it. **Never restart in the middle flow.**
```
Comment (markya0616) [inline README.md:36]: message -> messages

File: README.md (hunk 1 above)
Comment (cychuang0924) [reply README.md:34]: done

File: README.md (hunk 2 above)
Comment (cychuang0924) [reply README.md:35]: done

File: README.md (hunk 3 above)
Comment (cychuang0924) [reply README.md:36]: done

**Pull request merged**